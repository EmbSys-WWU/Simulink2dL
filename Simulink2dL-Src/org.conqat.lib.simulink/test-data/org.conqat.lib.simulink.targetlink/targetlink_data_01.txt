struct('comment',struct('text','','placement',1),'logdata',struct('maxnumvals',inf,'plotchannels',[],'loggingmode',3,'channels',1),'output',struct('variable','TMEN_TscRsccPri00','description',['Eingang vom Fahrpedal;' char(10) '' char(10) 'Drehzahlreglermode der primären Anforderung an den Motor (Requested Speed Control Conditions)' char(10) '' char(10) 'Für Drehzahlanforderungen und Begrenzungen der Drehzahlbeschleunigung wird ein Drehzahlreglermode vorgegeben. Dieser besagt, welche Reglerparameter des Drehzahlreglers verwendet werden sollen.' char(10) '' char(10) 'Hier gelten die Festlegungen laut SAE J1939.'],'name','$D','width',1,'address','','max',NaN,'min',NaN,'class','FCN_OUT','type','uint8','scaling','./LocalScaling','lsb',1,'offset',0,'arb',0,'freeze',1,'unit','','autoscalingmode',1,'lowerheadroom',1,'upperheadroom',1,'headroomunit','bit','keepdatatype',1,'checkmin',0,'checkmax',0,'castoutput',0,'show',0),'message',struct('name','','component',''),'virtual',1,'scalingvalid',0,'date',732744.367268391)