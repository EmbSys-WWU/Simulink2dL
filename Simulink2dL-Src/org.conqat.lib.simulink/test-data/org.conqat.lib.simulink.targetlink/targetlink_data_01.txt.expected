struct (
  'comment' -> struct (
    'placement' -> '1'
    'text' -> ''
  )
  'logdata' -> struct (
    'channels' -> '1'
    'loggingmode' -> '3'
    'maxnumvals' -> 'inf'
    'plotchannels' -> '[]'
  )
  'message' -> struct (
    'component' -> ''
    'name' -> ''
  )
  'output' -> struct (
    'address' -> ''
    'arb' -> '0'
    'autoscalingmode' -> '1'
    'castoutput' -> '0'
    'checkmax' -> '0'
    'checkmin' -> '0'
    'class' -> 'FCN_OUT'
    'description' -> 'Eingang vom Fahrpedal; char(10)  char(10) Drehzahlreglermode der prim�ren Anforderung an den Motor (Requested Speed Control Conditions) char(10)  char(10) F�r Drehzahlanforderungen und Begrenzungen der Drehzahlbeschleunigung wird ein Drehzahlreglermode vorgegeben. Dieser besagt, welche Reglerparameter des Drehzahlreglers verwendet werden sollen. char(10)  char(10) Hier gelten die Festlegungen laut SAE J1939.'
    'freeze' -> '1'
    'headroomunit' -> 'bit'
    'keepdatatype' -> '1'
    'lowerheadroom' -> '1'
    'lsb' -> '1'
    'max' -> 'NaN'
    'min' -> 'NaN'
    'name' -> '$D'
    'offset' -> '0'
    'scaling' -> './LocalScaling'
    'show' -> '0'
    'type' -> 'uint8'
    'unit' -> ''
    'upperheadroom' -> '1'
    'variable' -> 'TMEN_TscRsccPri00'
    'width' -> '1'
  )
  'date' -> '732744.367268391'
  'scalingvalid' -> '0'
  'virtual' -> '1'
)